backend/editor/entries.py

```diff
@@ -206,7 +225,13 @@ def update_node_children(entry, new_children_ids):
     existing_ids = [record['child.id'] for record in get_current_transaction().run(query, ids=list(added_children))]
     to_create = added_children - set(existing_ids)
 
-    for child in to_create:
+    #Normalising new children node ID
+    normalised_added_children = set()
+    for child_node in to_create:
+        child_language_code, new_child_name = child_node.split(":",1)
+        normalised_added_children.add(normalizing(new_child_name,child_language_code))
+
+    for child in normalised_added_children:
         main_language_code = child.split(":", 1)[0]
         create_node("ENTRY", child, main_language_code)

```

Comment:
I propose to instead do 

```suggestion
    normalised_added_children = []
    for child in to_create:
        main_language_code = child.split(":", 1)[0]
        created_node = create_node("ENTRY", child, main_language_code)
        normalised_added_children.append(created_node.id)
```

By the way we can use a shorter name like created_ids instead of normalised_added_children